### ISSUE TITLE
2.X `spacy.blank()` fails with "AttributeError: 'EntryPoints' object has no attribute 'get'"

### ISSUE THREAD

**Issue by convoliution** [explosion/spaCy#11576]
Forgive me if spaCy 2.X is no longer supported, but I at least wanted to file this in case others encounter the same issue.

This is occurring because

* `spacy==2.3.7` [depends on](https://github.com/explosion/spaCy/blob/v2.3.7/requirements.txt#L9) `catalogue<1.1.0`
* `catalogue==1.0.0` [depends on](https://github.com/explosion/catalogue/blob/v1.0.0/setup.cfg#L34) `importlib_metadata>=0.20`
* as noted in https://github.com/explosion/spaCy/pull/11569, `importlib_metadata` released a new major version over the weekend which shipped API breakages, including code being used in the `spacy.blank()` path.

## How to reproduce the behaviour

In a clean environment:

```sh
python -m pip install 'spacy<3.0'
python -c 'import spacy; spacy.blank("fr")'
```

## Your Environment

* Operating System: macOS 12.6
* Python Version Used: 3.7
* spaCy Version Used: 2.3.7
* Environment Information: only spaCy and its dependencies

**Comment by convoliution** [comment]
This can be worked around on a user's end by downgrading `importlib_metadata`:

```sh
python -m pip install 'importlib_metadata<5.0'
```

**Comment by polm** [comment]
Thanks for the detailed report! We don't actively work on v2 any more, but it does still see some use, so this is helpful to know.

**Comment by adrianeboyd** [comment]
Please try upgrading to the most recent compatible release of `catalogue` (`pip install catalogue==1.0.1`) to fix this problem.

Or if you install `spacy<3.0` in a new venv, you should automatically get the updated version of `catalogue`.

**Comment by liuverta** [comment]
> Or if you install spacy<3.0 in a new venv, you should automatically get the updated version of catalogue.

I can confirm this issue has been resolved for clean installs of `spacy<3.0`! Thank you so much @adrianeboyd for that patch to `catalogue` 🙇
